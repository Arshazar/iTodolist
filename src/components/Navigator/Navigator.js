import React from 'react'
import FilterButton from '../../containers/FilterButton'
import { VisibilityFilters } from '../../actions'
import { StyledNavigator } from './Navigator.styled'

const Navigator = () => (
    <StyledNavigator>
        <FilterButton filter={VisibilityFilters.SHOW_ALL}>All</FilterButton>
        <FilterButton filter={VisibilityFilters.SHOW_ACTIVE}>
            Todos
        </FilterButton>
        <FilterButton filter={VisibilityFilters.SHOW_COMPLETED}>
            Completed
        </FilterButton>
    </StyledNavigator>
)

export default Navigator
