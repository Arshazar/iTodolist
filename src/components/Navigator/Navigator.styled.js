import styled from 'styled-components'

export const StyledNavigator = styled.div`
    margin: 0;
    backface-visibility: hidden;

    button {
        border: 0;
        border-radius: 0;
        padding: 12px;
        color: white;
        background-color: ${({ theme }) => theme.primaryButtonColour};

        &:active {
            background-color: navy;
        }

        &:hover {
            background-color: navy;
        }
    }
`
