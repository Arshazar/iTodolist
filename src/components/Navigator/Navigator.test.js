import React from 'react'
import { shallow } from 'enzyme'
import Navigator from './Navigator'

it('renders navigator component', () => {
    expect(shallow(<Navigator />)).toMatchSnapshot()
})
