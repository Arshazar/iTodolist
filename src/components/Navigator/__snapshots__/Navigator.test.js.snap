// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navigator component 1`] = `ShallowWrapper {}`;
